static/app/components/guidedSteps/guidedSteps.spec.tsx

```diff
@@ -1,99 +1,128 @@
 import {useState} from 'react';
+import {RouterFixture} from 'sentry-fixture/routerFixture';
 
 import {render, screen, userEvent, within} from 'sentry-test/reactTestingLibrary';
 
 import {Button} from 'sentry/components/core/button';
 import {GuidedSteps} from 'sentry/components/guidedSteps/guidedSteps';
+import {useNavigate} from 'sentry/utils/useNavigate';
 
-describe('GuidedSteps', function () {
-  it('can navigate through steps and shows previous ones as completed', async function () {
-    render(
-      <GuidedSteps>
-        <GuidedSteps.Step stepKey="step-1" title="Step 1 Title">
-          This is the first step.
-          <GuidedSteps.StepButtons />
-        </GuidedSteps.Step>
-        <GuidedSteps.Step stepKey="step-2" title="Step 2 Title">
-          This is the second step.
-          <GuidedSteps.StepButtons />
-        </GuidedSteps.Step>
-        <GuidedSteps.Step stepKey="step-3" title="Step 3 Title">
-          This is the third step.
-          <GuidedSteps.StepButtons />
-        </GuidedSteps.Step>
-      </GuidedSteps>
-    );
+jest.mock('sentry/utils/useNavigate');
```

Comment:
@scttcper is actually in the process of banning hook mocks in tests, I'd suggest using `enableRouterMocks: false` instead